src/OmniSharp.DotNetTest/TestFrameworks/XunitTestFramework.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace OmniSharp.DotNetTest.TestFrameworks
+{
+    public class XunitTestFramework : TestFramework
+    {
+        public override string FeatureName { get; } = "XunitTestMethod";
+        public override string Name { get; } = "xunit";
+        public override string MethodArgument { get; } = "-method";
+
+        protected override bool IsTestAttributeName(string typeName)
+        {
+            return typeName == "Xunit.FactAttribute";
```

Comment:
Nope. `Xunit.TheoryAttribute` inherits from `Xunit.FactAttribute`: https://github.com/xunit/xunit/blob/master/src/xunit.core/TheoryAttribute.cs. Note that this didn't change any logic from before (just moved around) which already supported theories.